test/test_translator.py

```diff
@@ -138,8 +136,73 @@ def test_parse_polyline() :
         ]
     assert  test_coordinates == valid_coordinates
 
+def test_parse_polyline_empty_string() :
+
+    test_polyline= None
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    valid_coordinates= []
```

Comment:
This naming is a bit misleading. These aren't valid coordinates exactly, they are just the expected coordinates. Can we rename this to 'expected_coordinates'?

Reply:
i renamed it to 
    expected_coordinates= []
    assert  test_coordinates == expected_coordinates

Reply:
Nice, let's apply this same thinking to the rest of the unit tests as well.

Reply:
> Can we split `test_get_event_status()` out to it's three separate units of test? This is a great show of testing the full functionality in that method, but we'll want them to be separate tests.

I splitted into separate unit test